### PR TITLE
to_jwe/from_jwe fix

### PR DESCRIPTION
**Pull request by rohe**
Thanks for contributing to this library! Please include the following check
list in your pull request submission (you can delete this message). If your
changes don't need a change log or documentation update, please ignore this.

- [x ] Any changes relevant to users are recorded in the `CHANGELOG.md`.

to_jwe/from_jwe using elleptic curve crypto is dependent on pyjwkest to be of version 1.3.3 or later to work.

### CODE REVIEW

**Comment by rohe** [comment]
Actually pyjwkest 1.3.4 is better :-)

**Comment by rohe** [comment]
Not really sure why Tarvis fails. Works on my machine.

**Comment by lwm** [comment]
If you're relying on the new pyjwktest, it's being installed from the [requirements file](https://github.com/OpenIDC/pyoidc/blob/master/requirements/base.txt#L14). Try running a `make upgrade` and adding it to the PR.

We should probably have a discussion on how we want to support requirement.txt vs. setup.py installs. But for another day :)

**Comment by rohe** [comment]
Damn, OK got it.

**Comment by rohe** [comment]
OK, so the problem was with pyjwkest. Fixed that now.

File: tests/test_oauth2_message.py [hunk 1]
```diff
@@ -595,3 +605,33 @@ def test_missing_required(self):
 
         with pytest.raises(MissingRequiredAttribute):
             err.to_urlencoded()
+
+
+def test_to_jwt_rsa():
+    msg = Message(a='foo', b='bar', c='tjoho')
```
Comment (lwm) [inline tests/test_oauth2_message.py]: This can be a fixture that goes in `conftest.py`.

**Comment by lwm** [comment]
LGTM.

**Pull request merged**